Hazel/src/Platform/OpenGL/OpenGLVertexArray.cpp

```diff
@@ -54,11 +54,12 @@ namespace Hazel {
 		vertexBuffer->Bind();
 
 		uint32_t index = 0;
+	
```

Comment:
Last one, remove this empty line, wasn't there before (just looked at the diff), but looks indeed good to merge!